### PR TITLE
[NFC] Make some coding modifications

### PR DESCRIPTION
**Pull request by amdrexu**
1. Move CreateGetWaveSize to correct declaration place.
2. Change "llpc." to "lgc." in the comments and docs to match the reality.

Change-Id: I0da6cd060099a57dda67508ddec3794487daabbf

### CODE REVIEW

**Comment by amdvlk-admin** [comment]
### Test summary for commit 625d4b3f0e7ac687b5fac9974d38d5284c86a326
<details><summary>Driver commits used in build</summary><ul>

<li>CWPACK: amd-master 7387247eb9889ddcabbc1053b9c2052e253b088e</li>
<li>METROHASH: amd-master 3c566dd9cda44ca7fd97659e0b53ac953f9037d2</li>
<li>PAL: dev 44abc48d199dd28369bf7415eee1751c4bf616d2</li>
<li>SPVGEN: dev 580ade71d630a304805e9468ef22ad263c37d137</li>
<li>XGL: dev 38aa59bcf134d1cb340ddd2459b34b6eda798abe</li>
<li>LLVM-PROJECT: amd-gfx-gpuopen-dev c6950e03a4c97beadc88a8bf5bea3dc5aa2649ee</li>
</ul></details>

<details><summary>CTS tests (Failed: 0/225376)</summary><ul><li>Built with version 1.2.5.2</li>
<details open><summary>Rhel 8.2, Gfx10</summary><ul>

<li>Passed: 32512/56344 (57.7%)</li>
<li>Failed: 0/56344 (0.0%)</li>
<li>Not Supported: 23832/56344 (42.3%)</li><li>Warnings: 0/56344 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 18.04, Gfx9</summary><ul>

<li>Passed: 32352/56344 (57.4%)</li>
<li>Failed: 0/56344 (0.0%)</li>
<li>Not Supported: 23991/56344 (42.6%)</li><li>Warnings: 1/56344 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 20.04, Gfx8</summary><ul>

<li>Passed: 32688/56344 (58.0%)</li>
<li>Failed: 0/56344 (0.0%)</li>
<li>Not Supported: 23656/56344 (42.0%)</li><li>Warnings: 0/56344 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 20.04, Gfx103</summary><ul>

<li>Passed: 35669/56344 (63.3%)</li>
<li>Failed: 0/56344 (0.0%)</li>
<li>Not Supported: 20675/56344 (36.7%)</li><li>Warnings: 0/56344 (0.0%)</li>
</ul></details>


</ul></details>

File: lgc/state/PipelineState.cpp [hunk 1]
```diff
@@ -1180,6 +1180,10 @@ unsigned PipelineState::getMergedShaderWaveSize(ShaderStage stage) {
   }
 }
 
+// =====================================================================================================================
+// Get subgroup size for the specified shader stage.
+//
+// @param stage : Shader stage
```
Comment (kuhar) [inline lgc/state/PipelineState.cpp:1186]: `@returns` is missing

**Review by linqun — APPROVED** [review]

File: lgc/state/PipelineState.cpp (hunk 1 above)
Comment (amdrexu) [reply lgc/state/PipelineState.cpp:1186]: Thanks. Fixed.

**Review by kuhar — APPROVED** [review]

**Review by linqun — APPROVED** [review]

**Comment by amdvlk-admin** [comment]
### Test summary for commit 3eb5cc7f7b307e17def79f41a26505fc18cbe105
<details><summary>Driver commits used in build</summary><ul>

<li>CWPACK: amd-master 7387247eb9889ddcabbc1053b9c2052e253b088e</li>
<li>METROHASH: amd-master 3c566dd9cda44ca7fd97659e0b53ac953f9037d2</li>
<li>PAL: dev 44abc48d199dd28369bf7415eee1751c4bf616d2</li>
<li>SPVGEN: dev 580ade71d630a304805e9468ef22ad263c37d137</li>
<li>XGL: dev 38aa59bcf134d1cb340ddd2459b34b6eda798abe</li>
<li>LLVM-PROJECT: amd-gfx-gpuopen-dev c6950e03a4c97beadc88a8bf5bea3dc5aa2649ee</li>
</ul></details>

<details><summary>CTS tests (Failed: 0/225376)</summary><ul><li>Built with version 1.2.5.2</li>
<details open><summary>Rhel 8.2, Gfx10</summary><ul>

<li>Passed: 32512/56344 (57.7%)</li>
<li>Failed: 0/56344 (0.0%)</li>
<li>Not Supported: 23832/56344 (42.3%)</li><li>Warnings: 0/56344 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 18.04, Gfx9</summary><ul>

<li>Passed: 32352/56344 (57.4%)</li>
<li>Failed: 0/56344 (0.0%)</li>
<li>Not Supported: 23991/56344 (42.6%)</li><li>Warnings: 1/56344 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 20.04, Gfx8</summary><ul>

<li>Passed: 32688/56344 (58.0%)</li>
<li>Failed: 0/56344 (0.0%)</li>
<li>Not Supported: 23656/56344 (42.0%)</li><li>Warnings: 0/56344 (0.0%)</li>
</ul></details>

<details open><summary>Ubuntu 20.04, Gfx103</summary><ul>

<li>Passed: 35669/56344 (63.3%)</li>
<li>Failed: 0/56344 (0.0%)</li>
<li>Not Supported: 20675/56344 (36.7%)</li><li>Warnings: 0/56344 (0.0%)</li>
</ul></details>


</ul></details>

**Pull request merged**